starter-code/iteration-3/main.ts

```diff
@@ -1,7 +1,17 @@
 // Add the reference to the interface
 
 // Create class TodoItem that implements the corresponding interface
+class TodoItem implements TodoItemInterface {
 
+  constructor (
+    public name: string,
```

Comment:
You can also give these a starting value  if that is what you want when the class is instanced: `public name: string = "Cow"`